shared/charts/ssl-cert-check/templates/cronjob.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "ssl-cert-check.name" . }}
+  labels:
+    app: {{ template "ssl-cert-check.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: "{{ .Values.cronJobSchedule }}"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  concurrencyPolicy: Replace
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: {{ .Values.activeDeadlineSeconds }}
+      template:
+        metadata:
+          labels:
+            app: {{ template "ssl-cert-check.name" . }}
+            release: {{ .Release.Name }}
+        spec:
+          containers:
+          - name: ssl-cert-check
+            image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+            imagePullPolicy: {{ .Values.image.pullPolicy }}
+            env:
+              - name: STACKDRIVER_CLIENT
+                value: "true"
+              # Without sleep Stackdriver client some times fails to fetch
+              # app-default credentials on first try.
```

Comment:
This seems like a bug that should be resolved in the `ssl-cert-check` image and not here?

Reply:
I am not sure what is causing this, and it is pretty hard to debug, since the issue is intermittent, but yes, `ssl-cert-check` already has proper retry mechanics to deal with it, so this can be removed.